modules/gapi/perf/cpu/gapi_imgproc_perf_tests_cpu.cpp

```diff
@@ -134,6 +134,18 @@ INSTANTIATE_TEST_CASE_P(CannyPerfTestCPU, CannyPerfTest,
         Values(true, false),
         Values(cv::compile_args(IMGPROC_CPU))));
 
+INSTANTIATE_TEST_CASE_P(GoodFeaturesPerfTestCPU, GoodFeaturesPerfTest,
+    Combine(Values(AbsExact().to_compare_f()),
+        Values(CV_8UC1),
+        Values(szVGA, sz720p, sz1080p),
+        Values(3, 0),
+        Values(0.5, 0.01),
+        Values(3.0, 30.0),
+        Values(3, 9),
+        Values(true, false),
+        Values(0.04, 0.5),
```

Comment:
Do we need so many values in the performance test?

Reply:
quantity decreased